src/VisualStudio/Core/Test/CodeModel/AbstractCodeElementTests`1.vb

```diff
@@ -428,16 +428,25 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel
             Throw New NotSupportedException
         End Sub
 
-        Protected Sub TestPropertyDescriptors(code As XElement, ParamArray expectedPropertyNames As String())
+        ''' <summary>
+        ''' This function validates that our Code DOM elements are exporting the proper default interface.  That
+        ''' interface dictates what values are returned from methods like <see cref="ComponentModel.TypeDescriptor.GetProperties(Object)"/>.
+        '''
+        ''' It's not possible for those methods to be called in all environments where we test because it 
+        ''' requires type libraries, like EnvDTE, to be manually registered.  Testing for the ComDefaultInterface 
+        ''' attribute is an indirect way of verifying the same behavior.
+        ''' </summary>
+        ''' <typeparam name="TInterface">The default interface the Code DOM element should be exporting</typeparam>
+        ''' <param name="code">Code to create the Code DOM element</param>
+        Protected Sub TestPropertyDescriptors(Of TInterface As Class)(code As XElement)
```

Comment:
Should this now be renamed to TestDefaultInterface, since it now says nothing about properties?